docs/tutorials/lattice-fermions.ipynb

```diff
@@ -0,0 +1,762 @@
+{
```

Comment:
<p>no plots? :(</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
<p>I have added a plot now, and also another wave function to be complete... I think we can merge</p>